lib/config_loader.rb

```diff
@@ -0,0 +1,16 @@
+class ConfigLoader
+  def initialize(config_file)
+    @config =  YAML.load(File.open(config_file, 'r'))
+  end
+  def account_sid
+    fail "Account SID Missing" if !@config.has_key?('Account SID')
+    @config['Account SID']
+  end
+  def auth_token
+    fail "Auth Token Missing" if !@config.has_key?('Auth Token')
```

Comment:
Style/DeprecatedHashMethods: `Hash#has_key?` is deprecated in favor of `Hash#key?`. (https://github.com/bbatsov/ruby-style-guide#hash-key)<br>Style/NegatedIf: Favor `unless` over `if` for negative conditions. (https://github.com/bbatsov/ruby-style-guide#unless-for-negatives)